### PR TITLE
Move `Main` and `LaunchScreen` storyboards outside of `Base.lproj`

### PR DESCRIPTION
**Pull request by mokagio**
**TL;DR**: Our new localization tooling doesn't deal well with `Base.lproj` with no `.strings` in it. These files don't need to live in `Base.lproj` because they are not localized, so it's simpler to extract them. For what is wroth, this is the same setup as WordPress iOS.

---

### Description

The new location is `WooCommerce/Classes/ViewRelated/` for _both_. Previously, `Main.storyboard` was there, but `LaunchScreen.storyboard` was in the `Resources/` folder. That seemed inconsistent in particularafter the move away from the `Base.lproj` approach in this commit.

Xcode uses `Base.lproj` as the folder for the "development language". From the docs:

> The `Base.lproj` folder contains all the `.storyboard` or `.xib` files in the development language. The folders for languages that you add to your project contain a strings file for each `.storyboard` or `.xib` file in your
> project. The development language folder doesn’t contain strings files for the `.storyboard` and `.xib` files because they don’t require translation in the development language. All the language folders contain a `InfoPlist.strings` file used to localize bundle properties, such as the app name. Any other localized resources—such as strings files that you generate and use from your code—appear in these language folders.
>
> For example, for a universal iOS app that uses English as the development language, the `Base.lproj` folder contains
> `Main_iPad.storyboard` and `Main.iPhone.storyboard,` and the `en.lproj` folder contains `InfoPlist.strings.` If you add the German language, Xcode creates a `de.lproj` folder containing a `InfoPlist.strings`, `Main_iPad.strings`, and `Main_iPhone.strings` file. The files in the `de.lproj` folder contain placeholder text that needs to be translated to German.

The way I read that documentation is that we should have files in the `.storyboard` and `.xib` files in `Base.lproj` only if we plan to localize them. But in our case `LaunchScreen.storyboard` only shows the Woo logo image and `Main.storyboard` is an empty container. That is, we can safely move them away from `Base.lproj` and not have that affect the app.

The reason I noticed this is that the new localization automation cannot cope with an `.lproj` folder (in this case, `Base.lproj`) where it can't find a `Localizable.strings` file.

One alternative option would be to update our tooling to selectively skip `Base.lproj`, since the Apple docs state it shouldn't contain `.strings` files. That's an option, but in the meantime, this change would allow us to move on and get closer to finishing the current round localization tooling improvements.

### Testing instructions

After clearing the `DerivedData` folder, I run the app and verified both launch screen and the view controller in the app loaded.

<table>
<tr>
	<td>Screen Shot 2022-04-29 at 6 32 22 pm
</td>
<td>
Screen Shot 2022-04-29 at 6 45 19 pm
</td>
	<td>Screen Shot 2022-04-29 at 6 32 57 pm
</td>
</table>

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/LaunchScreen.storyboard [hunk 1]
```diff
@@ -25,12 +25,12 @@
                                 </constraints>
                             </imageView>
                         </subviews>
+                        <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
                         <color key="backgroundColor" name="WooCommercePurple60"/>
                         <constraints>
                             <constraint firstItem="Q7F-qQ-dHf" firstAttribute="centerX" secondItem="Ze5-6b-2t3" secondAttribute="centerX" id="Ilw-Kx-7ga"/>
                             <constraint firstItem="Q7F-qQ-dHf" firstAttribute="centerY" secondItem="Ze5-6b-2t3" secondAttribute="centerY" id="Wnf-iK-LGA"/>
                         </constraints>
-                        <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
```
Comment (jaclync) [inline WooCommerce/Classes/ViewRelated/LaunchScreen.storyboard:33]: these diffs from just opening the file in the Interface Builder always make me 🙄

**Review by jaclync — APPROVED** [review]
Changes LGTM :shipit: 

Everything in `Main.storyboard` should be localized within the view controller(s), so no problem there. For `LaunchScreen.storyboard`, I could see a chance where it makes more sense to show a localizable string in the launch screen instead of showing a static image. But we don't have any plans to do that, and we can always move the launch screen when we do. Let's enable the tooling first 👍 

Tested on a device with incremental build, and also a clean build on a simulator ✅

File: WooCommerce/Classes/ViewRelated/LaunchScreen.storyboard (hunk 1 above)
Comment (mokagio) [reply WooCommerce/Classes/ViewRelated/LaunchScreen.storyboard:33]: Very looking forward to the day we'll be past Interface Builder. Farewell old friend, you served us well back in the day, but haven't aged well 👋
